src/client/src/App.js

```diff
@@ -0,0 +1,26 @@
+import './App.css'
+const React = require('react')
+const HomePage = require('./components/homepage')
+const Register = require('./components/register')
+const ChatPage = require('./components/chatpage')
+const {BrowserRouter, Route} = require('react-router-dom')
+
+const LOGIN_URL = '/'
```

Comment:
I wonder if we could have something like routing JSON, where we will keep it and share among components?